apps/evisa/behaviours/agent-email.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+/* eslint-disable no-unused-vars, no-console */
+
+const hof = require('hof');
+const notify = hof.components.notify;
+// const { notify } = require('hof');   // hof.components.notify // TODO refactor to use this
+const path = require('path');
+
+const parse = (model, translate) => {
+  console.log('=====================');
+  console.log(model.images[0].url);
+  console.log('=====================');
```

Comment:
remember to take out the console logs :D

Reply:
Will do, they're coming in handy atm.